pandas/tseries/offsets.py

```diff
@@ -1160,6 +1162,196 @@ def onOffset(self, dt):
     _prefix = 'MS'
 
 
+class SemiMonthOffset(SingleConstructorOffset):
+    _adjust_dst = True
+    _default_day_of_month = 15
+    _min_day_of_month = 2
+
+    def __init__(self, n=1, normalize=False, **kwds):
+        self.day_of_month = kwds.get('day_of_month',
```

Comment:
I know a lot of the other offsets just use `**kwds`, but where you're defining a constructor, I think it would make sense to add `day_of_month` as an argument?